css/src/schema-blocks.css

```diff
@@ -16,3 +16,16 @@
 	border: #CCCCCC 1px solid;
 	border-radius: 0;
 }
+
+.yoast-job-block__salary .wp-block-yoast-job-salary-range {
+	max-width: 25%;
+}
+
+.yoast-job-block__salary .wp-block-yoast-job-salary-range input[type=number] {
+	-moz-appearance: textfield;
+}
+
+.editor-styles-wrapper .wp-block-yoast-job-salary [data-block] {
+	margin-top: 0;
+	margin-bottom: 0;
+}
```

Comment:
Since these styles are job posting related (a Premium feature) I would add them to [the Premium Schema blocks CSS](https://github.com/Yoast/wordpress-seo-premium/blob/trunk/assets/css/src/premium-schema-blocks.css) instead 🙂 .